apps/ui/src/components/OnboardingTask.vue

```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
+interface Task {
+  description: string;
+  link?: { name: string; params?: Record<string, any> };
+  externalLink?: string;
+  currentStep?: number;
+  totalSteps?: number;
+}
+
+defineProps<{ task: Task }>();
+</script>
+<template>
+  <div class="border-b mx-4 py-[14px] flex gap-x-2.5">
+    <IS-flag class="text-skin-link mt-1 shrink-0" />
+    <div>
+      <template v-if="task.link">
+        <AppLink :to="task.link">{{ task.description }}</AppLink>
+      </template>
+      <template v-else-if="task.externalLink">
+        <a :href="task.externalLink" target="_blank">{{ task.description }}</a>
+      </template>
+      <template v-else>
```

Comment:
We could merge `link` and `externalLink` together, and let `AppLink` do the magic

Reply:
`AppLink`  works perfectly ❤️